fish-pepper/run-java-sh/fp-files/run-java.sh

```diff
@@ -378,72 +241,9 @@ diagnostics_options() {
   fi
 }
 
-# Replicate thread ergonomics for tiered compilation.
-# This could ideally be skipped when tiered compilation is disabled.
-# The algorithm is taken from:
-# OpenJDK / jdk8u / jdk8u / hotspot
-# src/share/vm/runtime/advancedThresholdPolicy.cpp
-ci_compiler_count() {
-  local core_limit="$1"
-  local log_cpu=$(calc 'log2($1)' "$core_limit")
-  local loglog_cpu=$(calc 'log2(max2($1,1))' "$log_cpu")
-  local count=$(calc 'max2($1*$2,1)*3/2' "$log_cpu" "$loglog_cpu")
-  local c1_count=$(calc 'max2($1/3,1)' "$count")
-  local c2_count=$(calc 'max2($1-$2,1)' "$count" "$c1_count")
-  [ $(c2_disabled) = true ] && echo "$c1_count" || echo $(calc '$1+$2' "$c1_count" "$c2_count")
-}
-
-cpu_options() {
```

Comment:
Why removing all the CPU related options ? What is the alternative for Java 11 ?

Reply:
The CPU calculations are best let auto calculated by JVM or supplied using Container level limits (k8s/OpenShift). These were anyway required when auto calculations in JVM were not a feature for containers.